src/pages/ServiceDetails/ServiceMetrics.tsx

```diff
@@ -98,10 +102,20 @@ class ServiceMetrics extends React.Component<ServiceId, ServiceMetricsState> {
                   <li>Request duration: {this.histogram(this.state.requestDurationIn)}</li>
                   <li>Response size: {this.histogram(this.state.responseSizeIn)}</li>
                 </ul>
+
+                <SparklineChart
+                  id="line-chart-2"
```

Comment:
Since this is using an 'id' I assume this will only be used in one place in the application. There can only be one id of a given name in the application. Really cuts down on reusability

Reply:
This whole PR was not meant "for production", but rather to research what could be done and how.